public/js/applicantForm.js

```diff
@@ -16,13 +16,8 @@
 
     if (document.querySelector('#chosenCourseId')) {
         const chosenCourseId = document.querySelector('#chosenCourseId').dataset.selected;
```

Comment:
this variable isnt needed, but it could contain just the element selector in order to reduce DOM selection